api-tester/src/main/java/com/revenuecat/apitester/java/StoreTransactionAPI.java

```diff
@@ -13,7 +13,6 @@
 final class StoreTransactionAPI {
     static void check(final StoreTransaction transaction) {
         final String orderId = transaction.getOrderId();
-        final List<String> skus = transaction.getSkus();
```

Comment:
we've typically left deprecated stuff in the public APIs i think

Reply:
OOPS... this must have been from a bad rebase 😅